examples/temperature_convertor/soln.py

```diff
@@ -0,0 +1,29 @@
+
+def fahrenheit_to_celsius():
+    # get the fahrenheit temperature from the user
+    fahrenheit = float(input("Enter Temperature in Fahrenheit : "))
+    # use the formula to calculate temperature in celsius
+    celsius = (fahrenheit - 32) * 5.0/9.0
+    # output with appropriate units
+    print("Temperature:", fahrenheit, "F = ", celsius, "C")
```

Comment:
We'll first be teaching printing with string concatenation and explicitly casting ints to strs, like so:

```python
n = 42
print("n = " + str(n)) 
```

so we might want to be consistent with that :)